src/QualificationRunner/Program.cs

```diff
@@ -63,7 +63,7 @@ private static IOSPSuiteLogger initializeLogger(CLICommand runCommand)
          loggerCreator.AddLoggingBuilderConfiguration(x => x.SetMinimumLevel(runCommand.LogLevel).AddConsole());
 
          if (!string.IsNullOrEmpty(runCommand.LogFileFullPath))
-            loggerCreator.AddLoggingBuilderConfiguration(builder => builder.AddFile(runCommand.LogFileFullPath, runCommand.LogLevel));
+            loggerCreator.AddLoggingBuilderConfiguration(builder => builder.AddFile(runCommand.LogFileFullPath, runCommand.LogLevel, true));

```

Comment:
Let's name the parmaeter `share: true` so that we know what this is. The other parameters are self explainatory